src/Headless/Avalonia.Headless/HeadlessWindowExtensions.cs

```diff
@@ -43,14 +43,30 @@ public static class HeadlessWindowExtensions
     /// <summary>
     /// Simulates keyboard press on the headless window/toplevel.
     /// </summary>
+    [Obsolete("Use the overload that takes a physical key and key symbol instead.")]
     public static void KeyPress(this TopLevel topLevel, Key key, RawInputModifiers modifiers) =>
-        RunJobsOnImpl(topLevel, w => w.KeyPress(key, modifiers));
+        KeyPress(topLevel, key, modifiers, PhysicalKey.None, null);
+
+    /// <summary>
+    /// Simulates keyboard press on the headless window/toplevel.
+    /// </summary>
+    public static void KeyPress(this TopLevel topLevel, Key key, RawInputModifiers modifiers, PhysicalKey physicalKey,
+        string? keySymbol) =>
+        RunJobsOnImpl(topLevel, w => w.KeyPress(key, modifiers, physicalKey, keySymbol));
```

Comment:
Can't we loosely map from Key to PhysicalKey and keySymbol? For many headless tests developers would just assume English QWERTY, and there is no need to obsolete old method. But new one is needed anyway, yes.

Reply:
We can, but since all implementations (Android excepted - pending) now provide the physical key, I thought it would be best for it to be always specified rather than incorrectly loosely mapped from the virtual key (controls and users then can't rely on it). 

In a perfect world, I'd like users to avoid `Key` if possible so we can cleanup/replace it in the distant future (realistically I know it probably won't happen).

That said, the main usage here is unit tests, where the keyboard layout doesn't really matter, as you mentioned. Would you accept an overload taking only a `PhysicalKey` that translates to QWERTY, while being explicit about it? e.g. `KeyPressQwerty(TopLevel, PhysicalKey, RawInputModifiers)`. We already have the mapping public in this direction.